src/test/java/test/integration/AddArquillianAlgeronConsumerTest.java

```diff
@@ -0,0 +1,87 @@
+package test.integration;
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.forge.addon.projects.Project;
+import org.jboss.forge.addon.projects.ProjectFactory;
+import org.jboss.forge.addon.projects.facets.DependencyFacet;
+import org.jboss.forge.addon.ui.controller.CommandController;
+import org.jboss.forge.addon.ui.result.Failed;
+import org.jboss.forge.addon.ui.result.Result;
+import org.jboss.forge.addon.ui.test.UITestHarness;
+import org.jboss.forge.arquillian.command.AddAlgeronConsumerDependenciesCommand;
+import org.jboss.forge.arquillian.command.AddArquillianCommand;
+import org.jboss.forge.arquillian.command.AddTestFrameworkCommand;
+import org.jboss.forge.arquillian.container.model.ContractConsumerLibrary;
+import org.jboss.forge.arquillian.testframework.algeron.AlgeronConsumer;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.inject.Inject;
+
+@RunWith(Arquillian.class)
+public class AddArquillianAlgeronConsumerTest
+{
+
+   @Inject
+   private UITestHarness testHarness;
+
+   @Inject
+   private ProjectFactory factory;
+
+   @Test
+   public void should_add_arquillian_algeron_consumer_dependencies() throws Exception {
+
+      Project project = factory.createTempProject();
+
```

Comment:
Then I get java.lang.IllegalStateException: Facet type [org.jboss.forge.arquillian.testframework.junit.JUnitFacet] could not be installed into [/var/folders/xd/_pglfgy97snfpv5pfd74qjtr0000gn/T/forgetemp208916886911978615] of type [org.jboss.forge.addon.maven.projects.MavenProject]. You may wish to check for inconsistent origin state as partial installation may have occurred.